controllers/commentController.js

```diff
@@ -0,0 +1,38 @@
+const {
+    addComment,
+    fetchComments,
+    removeComment
+} = require('../models/commentModel')
+
+exports.getComments = (req, res, next) => {
+    const {review_id} = req.params
+
+    fetchComments(review_id)
+    .then(comments => {
+        res.status(200).send({comments})
+    })
+    .catch(err => next(err))
+}
+
+exports.postComment = (req, res, next) => {
+    const {review_id} = req.params
+    const {username, body} = req.body
+
+    if (!username || !body) next({statusCode: 400, msg: "Patch body must contain a username and body text for the comment"})
+    addComment(review_id, username, body)
+    .then(comment => {
```

Comment:
You should either use predominantly async/await or promise syntax, instead of mixing/matching. Make your code consistent